Add a fixedfooter placement style (#121)
readthedocs/ethical-ad-client: docs/index.rst

   <div data-ea-publisher="..." data-ea-type="image" data-ea-style="stickybox"></div>

Screenshot
``````````

.. image:: _static/stickybox.gif
    :align: center
    :width: 100%

FixedFooter
~~~~~~~~~~~

.. versionadded:: 1.6

> The "FixedFooter" style is a floating placement is a text-only placement
> attached to the bottom of the screen.
> By floating, it ensures that the ad is always seen
> (and therefore results in billed views that make the publisher money).

Reviewer: ```suggestion
The "FixedFooter" style is a floating, text-only placement
attached to the bottom of the screen.
By floating, it ensures that the ad is always seen
(resulting in the highest view rate, generating the most revenue).
```
